src/test/java/edu/kis/powp/jobs2d/events/SelectLoadEditedCommandOptionListener.java

```diff
@@ -0,0 +1,39 @@
+package edu.kis.powp.jobs2d.events;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+import edu.kis.powp.jobs2d.command.*;
+import edu.kis.powp.jobs2d.command.manager.DriverCommandManager;
+import edu.kis.powp.jobs2d.features.CommandsFeature;
+
+public class SelectLoadEditedCommandOptionListener implements ActionListener {
+
+    @Override
+    public void actionPerformed(ActionEvent e) {
+        ComplexCommandEditor editor = new ComplexCommandEditor();
+        editor.addCommand(new SetPositionCommand(-20, -50));
+        editor.addCommand(new OperateToCommand(-20, -50));
+        editor.addCommand(new SetPositionCommand(-20, -40));
+        editor.addCommand(new OperateToCommand(-20, 50));
+        editor.addCommand(new SetPositionCommand(0, -50));
+        editor.addCommand(new OperateToCommand(0, -50));
+        editor.addCommand(new SetPositionCommand(0, -40));
+        editor.addCommand(new OperateToCommand(0, 50));
+        editor.addCommand(new SetPositionCommand(70, -50));
+        editor.addCommand(new OperateToCommand(20, -50));
+        editor.addCommand(new OperateToCommand(20, 0));
+        editor.addCommand(new OperateToCommand(70, 0));
+        editor.addCommand(new OperateToCommand(70, 50));
+        editor.addCommand(new OperateToCommand(20, 50));
+
+        editor.addCommand(new OperateToCommand(100, 100));
```

Comment:
Code duplication:
https://github.com/iis-powp-2025/powp_jobs2d_projects/blob/master/src/test/java/edu/kis/powp/jobs2d/events/SelectLoadSecretCommandOptionListener.java
Please extract to a factory method in main.